js/keybindings.js

```diff
@@ -353,15 +353,10 @@ settings.get('keyMap', function (keyMapSettings) {
   defineShortcut('switchToNextTask', function (d) {
     taskOverlay.show()
 
-    var currentTaskIdx = tasks.get().map(function (task) {
-      return task.id
-    }).indexOf(currentTask.id)
+    const currentTaskIdx = tasks.indexOf(currentTask)
 
-    if (tasks.get()[currentTaskIdx + 1]) {
-      browserUI.switchToTask(tasks.get()[currentTaskIdx + 1].id)
-    } else {
-      browserUI.switchToTask(tasks.get()[0].id)
-    }
+    const nextTaskIndex = (currentTaskIdx + 1) % tasks.getLength()
```

Comment:
It's neat that you can do this, but I'm not sure using it is a good idea. With the old way, it's pretty easy to tell that it's switching to the next task, or the first task if that one doesn't exist, whereas this way it takes a second to think through the math and figure out what it's doing. If we wanted to shorten this, something like this might work:

```
(tasks.byIndex(currentTaskIdx + 1) || tasks.byIndex(0)).id
```

Reply:
Addressed in 3486a1a.